manifests/pe_metric.pp

```diff
@@ -22,7 +22,7 @@
   }
 
   $config_hash = {
-    'hosts'              => $hosts,
+    'hosts'              => $hosts.sort(),
```

Comment:
Does this match the style guide or should it be `sort($hosts)`?

Reply:
I don't see anything in the style guide about it. Happy to change it.